crypto/engine/eng_cryptodev.c

```diff
@@ -213,29 +222,31 @@ static struct {
 static struct {
     int id;
     int nid;
-    int keylen;
+    int digestlen;
 } digests[] = {
+#  if 0
+        /* HMAC is not supported */
```

Comment:
Can this be guarded with something like `# ifdef CRYPTO_MD5_HMAC` and similar?  One guard for each item?